.changeset/tricky-rockets-attend.md

```diff
@@ -0,0 +1,14 @@
+---
+"@web5/common": patch
+"@web5/crypto": patch
+"@web5/crypto-aws-kms": patch
+"@web5/dids": patch
+"@web5/credentials": patch
+"@web5/agent": patch
+"@web5/user-agent": patch
+"@web5/proxy-agent": patch
+"@web5/identity-agent": patch
+"@web5/api": patch
```

Comment:
`@web5/api` changests must be included in a separate file from the other changests.

Not sure if it's in the CONTRIBUTING.md file, but it's something that emerged due to not always publishing the `@web5/api` package.

```suggestion
```

Reply:
Updated!